autoendpoint/src/routers/fcm/router.rs

```diff
@@ -203,6 +225,13 @@ mod tests {
         map
     }
 
+    fn gcm_router_data() -> HashMap<String, serde_json::Value> {
+        let mut map = HashMap::new();
+        map.insert("auth".to_string(), serde_json::to_value(FCM_TOKEN).unwrap());
```

Comment:
Per [gcm.py](https://github.com/mozilla-services/autopush/blob/5fd37e2/autopush/router/gcm.py#L69) we should expect auth/senderID to be nested under "creds".